libraries/script-engine/src/UsersScriptingInterface.h

```diff
@@ -39,9 +41,14 @@ class UsersScriptingInterface : public QObject, public Dependency {
     Q_PROPERTY(bool canKick READ getCanKick)
     Q_PROPERTY(bool requestsDomainListData READ getRequestsDomainListData WRITE setRequestsDomainListData)
 
+    Q_PROPERTY(unsigned int NO_BAN READ getNoBan CONSTANT)
+    Q_PROPERTY(unsigned int BAN_BY_USERNAME READ getBanByUsername CONSTANT)
+    Q_PROPERTY(unsigned int BAN_BY_FINGERPRINT READ getBanByFingerprint CONSTANT)
+    Q_PROPERTY(unsigned int BAN_BY_IP READ getBanByIP CONSTANT)
```

Comment:
I think everywhere else where you pass around an int, you should be passing around an unsigned int like these

Reply:
I guess I'm just copying how it is done in FilterFlags, though they may actually use unsigned ints *everywhere*. Is that a concern? If making it an int won't break anything then I don't see why I wouldn't.

Reply:
we should switch to unsigned int everywhere, otherwise I can imagine something weird happening with negative numbers or something